packages/voorbeeld-design-tokens/documentation/theme-builder/theme styling/voorbeeld-theme.ts

```diff
@@ -0,0 +1,13 @@
+export const fontFamilystep = {
+  '--utrecht-document-font-family': '"Comic Sans MS"',
+};
+
+export const commoncolorstep = {
```

Comment:
Ik zie dat de eerste const in camelCase is, de rest is met alleen kleine letters. 
Misschien fijn om dit gelijk te houden in het project?